src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -501,10 +501,10 @@ typedef struct struct_8085D200 {
     /* 0x9 */ u8 unk_9;
 } struct_8085D200; // size = 0xC
 
-f32 D_80862AFC; // distance of the analog stick to its center
-s16 D_80862B00; // analog stick angle/yaw
-s16 D_80862B02; // analog stick yaw + camera yaw
-s32 D_80862B04; // boolean, set to the return value of func_8083216C
+f32 sPlayerControlStickMagnitude; // distance of the analog stick to its center
+s16 sPlayerControlStickAngle;     // analog stick angle/yaw
```

Comment:
comments can prob go actually?
I think magnitude makes more sense overall than calling it a distance. And im not sure the angle is really a "yaw", it only becomes yaw after its applied to links angle in 3d space